.gitignore

```diff
@@ -1 +1,3 @@
 .env
```

Comment:
What are you storing in env?

Reply:
I was storing some settings I thought was required... but I figured another way out. Can remove this